ts/features/wallet/component/PaymentMethodCapabilities.tsx

```diff
@@ -36,7 +36,10 @@ const capabilityFactory = (
       return null;
     case EnableableFunctionsTypeEnum.BPD:
       return bpdEnabled ? (
-        <BpdPaymentMethodCapability paymentMethod={paymentMethod} />
+        <BpdPaymentMethodCapability
+          paymentMethod={paymentMethod}
+          key={EnableableFunctionsTypeEnum.BPD}
```

Comment:
It should avoid to use a const as key
This function should accept a key as parameter

Reply:
@Undermaken At the beginning I used the string of capability function since we should not have multiple elements with the same enum value, but you're right it's better to use an index, fixed in b4c101c